modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -121,6 +127,11 @@ template<typename Net> class Params {
         return *this;
     }
 
+    Params& pluginConfig(IEConfig cfg) {
+        desc.config = std::move(cfg);
+        return *this;
+    }
```

Comment:
I would : 
- add rvalue ref overload 
- changed this to accept  `const &` to copy parameter into (i.e. have one copy operation instead of copy + move)

Reply:
In case `rvalue` it can be moved to `pluginConfig` and then moved to desc.config

Reply:
You mean we can avoid move operation for `const T&`, got it, fixed